### PR TITLE
Add an aria-label attribute to the EmberConf link

### PR DESCRIPTION
**Pull request by paulgaumer**
Fix Issue #234 

Hi there,

I ran Lighthouse on your homepage and fixed one of the links raising the error "Links do not have a discernible name" (line 30)

The other errors come from links located in the footer, and strangely haven't been able to locate the footer's code in the repo (I am new to ember). Happy to fix them as well with a little guidance :)

### CODE REVIEW

**Comment by amyrlam** [comment]
@paulgaumer thank you for the PR! You have some template lint errors [here](https://travis-ci.org/ember-learn/ember-website/builds/595020855#L506-L528). Do you want to take a try at fixing them?

This is the linter: https://github.com/ember-template-lint/ember-template-lint, there are some useful extensions if you're using an editor like VSCode

File: app/templates/index.hbs [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 <div class="emberconf-announcement call-to-action__box">
   <div class="logo-and-date">
-    <a href="//emberconf.com/">
+    <a href="//emberconf.com/" aria-label="EmberConf 2020 Logo">
```
Comment (MelSumner) [inline app/templates/index.hbs]: It should just be `aria-label="Ember conf 2020"`

Comment (amyrlam) [reply app/templates/index.hbs]: @MelSumner updated this!

**Comment by amyrlam** [comment]
@paulgaumer fixed the unrelated linting errors. It's possible that you have something set in your editor that is auto-formatting? Will merge

**Pull request merged**